server/src/controllers/Account.controller.js

```diff
@@ -59,4 +62,38 @@ export default class AccountController {
       message: 'Account created'
     });
   }
+
+  /**
+ * @method getAccount
+ *
+ * @param {Object} req
+ * @param {Object} res
+ *
+ * @returns {object} status code and data message
+ */
+  static getAccount(req, res) {
+    const { accountNumber } = req.params;
+    const retriveAccount = accounts
+      .find(account => account.accountNumber === parseInt(accountNumber, 10));
```

Comment:
You need to update this logic to check that the account also belongs to the user. Another user shouldn't be able to get a user's account.

Reply:
done...